### ISSUE TITLE
Remove nfhout, nfhmax_hf, nfhout_hf etc configuration parameters

### ISSUE THREAD

**Issue by DusanJovic-NOAA** [NOAA-EMC/ufsatm#731]
History output times/frequency can be now configured using "output_fh" parameter in model_configure. It supports following options (for example)

output_fh: 1 -1 (gives hourly output frequency)
output_fh: 0.5 -1 (gives 30mins output frequency)
output_fh: 0.5 1.5 (output at fh=30mins and 1.5hours)

Old parameters used to define output times and/or frequency like nfhmax_hf, nfhout_hf etc. should be removed.

**Comment by DusanJovic-NOAA** [comment]
@junwang-noaa

**Comment by BinLiu-NOAA** [comment]
@DusanJovic-NOAA or @junwang-noaa, with the output_fh option,  is there an easy way (other than explicitly listing all the desired output hours) to specify saying hourly output before day 5 (120-h, e.g., 1 2 3 4 5 6 ... 120), and then 3-hourly output afterwards till the end of the forecast (e.g., 123 126 129 ...). Thanks!

**Comment by DusanJovic-NOAA** [comment]
As far as I know you can either specify the output frequency or explicit output time.

**Comment by junwang-noaa** [comment]
@BinLiu-NOAA I'd suggest to have a script level function (a list of frequencies/forecast time and the final output_fh array).

**Comment by BinLiu-NOAA** [comment]
> As far as I know you can either specify the output frequency or explicit output time.

@DusanJovic-NOAA, Thanks for your reply! It seems to me the old nfhout, nfhmax_hf and nfhout_hf method can easily achieve what I mentioned above, for example,
```
nfhout:                  3       # Output frequency in hours after forecast hour nfhmax_hf
nfhmax_hf:               120     # Number of forecast hours until output frequency nfhout takes affect
nfhout_hf:               1       # Output frequency in hours until forecast hour nfhmax_hf
```
I wish both the old nfhout/nfhmax_hf/nfhout_hf approach and the new output_fh approach can be kept. But, I guess it is properly too late. I saw the PR has been merged already.

**Comment by BinLiu-NOAA** [comment]
> @BinLiu-NOAA I'd suggest to have a script level function (a list of frequencies/forecast time and the final output_fh array).

@junwang-noaa, Yeah, that is doable. Just one more thing, is there any size limit for the output_fh item in model_configure? The output_fh item could be very long for applications like GFS (hourly output till day 5, then 3-hourly till day 15/16). Also need to consider GEFS and SFS output frequencies. Thanks!

**Comment by junwang-noaa** [comment]
@BinLiu-NOAA That is a good question. According to ESMF configuration variable,  the limit is 1024 characters. ESMF group has plan to update the ESMF_ConfigLoadFile line limit. @danrosen25

**Comment by danrosen25** [comment]
> @BinLiu-NOAA That is a good question. According to ESMF configuration variable, the limit is 1024 characters. ESMF group has plan to update the ESMF_ConfigLoadFile line limit. @danrosen25

I have implemented a line limitation fix for 8.6.1 and it needs testing: https://github.com/esmf-org/esmf/tree/bugfix/esmf_config
This will remove the line limitation but there will still be a maximum file size limitation of approximately 256,000 characters. If you expect to start loading files greater than 256 KB then let me know.

**Comment by DusanJovic-NOAA** [comment]
Fixed in #734, which is merged via #735
